packages/shared/components/molecules/TransactionDetails.svelte

```diff
@@ -174,6 +185,11 @@
                     {localize('pills.locked')}
                 </Pill>
             {/if}
+            {#if type === ActivityType.Alias}
+                <Pill backgroundColor="green-300" darkBackgroundColor="gray-200">
+                    {localize('pills.aliasCreated')}
+                </Pill>
+            {/if}
```

Comment:
We should move this logic inside of ActivityStatusPill and also deisplay text based on inclusion.

Reply:
will do on next PR